whistl/expression_qc.py

```diff
@@ -0,0 +1,42 @@
+'''This script looks at gene expression files and checks how much data is missing to find samples
+that should be removed from the analysis.'''
+
+import argparse
+import os
+
+import pandas as pd
+
+if __name__ == '__main__':
+    # Get arguments dir, num_genes
+    parser = argparse.ArgumentParser(description='''This script looks at gene expression files
+                                     and checks how much data is missing to find samples
+                                     that should be removed from the analysis.''')
+    parser.add_argument('study_dir', help='The path to the directory containing subdirectories '
+                                          'corresponding to studies')
+    parser.add_argument('missing_data_dir',
+                        help='The directory to which to move files that do not meet the threshold '
+                             'number of genes')
+    parser.add_argument('--num_genes',
+                        help='The number of genes without missing data required to '
+                             'avoid being flagged for further examination',
+                        type=int, default=15000)
+
+    args = parser.parse_args()
+
+    for dir_ in os.listdir(args.study_dir):
+        if 'SRP' in dir_ or 'GSE' in dir_:
+            study = dir_.strip('/')
```

Comment:
[os.path.normpath](https://docs.python.org/3/library/os.path.html#os.path.normpath) should give you what you want here (+ cross-platform compatibility), I think.

Reply:
Yep, you're right